core/ui/public/i18n/en/translation.json

```diff
@@ -1361,33 +1361,39 @@
     "syslog_forwarder": {
         "configure_export_to_syslog": "Configure export to Syslog",
         "export_to_syslog": "Export to Syslog",
-        "hostname_ip_address": "Hostname or IP address",
-        "port": "Port",
+        "hostname_ip_address": "Host name or IP address",
+        "port_number": "Port number",
         "protocol": "Protocol",
         "format": "Format",
+        "udp": "UDP",
+        "tcp": "TCP",
+        "rfc3164": "RFC 3164",
+        "rfc5424": "RFC 5424",
+        "port_placeholder": "514",
         "syslog_export_enabled": "Export to Syslog enabled",
         "syslog_export_failed": "Export to Syslog failed",
-        "export_starting_date_tooltip": "Lorem Ipsum",
-        "format_tooltip": "Lorem Ipsum",
-        "start_date": "Start date",
-        "start_time": "Start time (24-hour)"
+        "start_date": "Start date for export",
+        "start_time": "Start time (24-hour)",
+        "export_starting_date": "Export start date",
+        "use_last_timestamp": "Use last timestamp ({timestamp})",
+        "choose_a_datetime": "Choose date and time"
     },
     "cloud_log_manager_forwarder": {
         "configure_export_to_cloud_log_manager": "Configure Cloud Log Manager",
         "export_to_cloud_log_manager": "Export to Cloud Log Manager",
         "address": "Cloud Log Manager URL",
-        "tenant": "Tenant",
+        "tenant": "Company Unique Key",
         "export_starting_date": "Export start date",
         "use_last_timestamp": "Use last timestamp ({timestamp})",
         "choose_a_datetime": "Choose date and time",
-        "tenant_tooltip": "Lorem Ipsum",
-        "export_starting_date_tooltip": "Lorem Ipsum",
+        "tenant_tooltip": "This key, also known as Tenant, identifies and associates the cluster logs within a Company in Cloud Log Manager. You can find it under Users and Companies > Companies.",
         "cloud_log_manager_export_enabled": "Export to Cloud Log Manager enabled",
         "cloud_log_manager_export_failed": "Export to Cloud Log Manager failed",
-        "start_date": "Start date",
+        "start_date": "Start date for export",
         "start_time": "Start time (24-hour)",
-        "more_details": "More details",
-        "available_with_subscription_title": "Available with subscription",
-        "available_with_subscription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+        "learn_more": "Learn more",
+        "address_placeholder": "https://nar.nethesis.it",
+        "available_with_subscription_title": "Activate Subscription to enable this feature",
```

Comment:
```suggestion
        "available_with_subscription_title": "Activate subscription to enable this feature",
```

Reply:
IMO here we are referring to the concept of "subscription", not the actual page or card in the UI. Anyway it's just a detail